src/manipulateToDo.js

```diff
@@ -0,0 +1,305 @@
+    classArray.includes('fa-check')
+    || classArray.includes('checkbox')
+    || classArray.includes('to-do')
+  ) {
+    elUnder = elUnder.parentElement.parentElement;
+  } else if (
+    classArray.includes('fa-arrows-alt')
+    || classArray.includes('two')
+  ) {
+    elUnder = elUnder.parentElement;
+  }
+  elUnder.style.backgroundColor = '#4c3c3c';
+  elUnder.style.color = '#fff';
+  Array.from(elUnder.childNodes).forEach((node) => {
+    if (node.nodeName === 'DIV') {
+      const nodes = node.childNodes;
+      Array.from(nodes).forEach((one) => {
+        one.style.backgroundColor = '#4c3c3c';
+        one.style.color = '#fff';
+      });
+    }
+    node.style.backgroundColor = '#4c3c3c';
+    node.style.color = '#fff';
+  });
+};
+
+const drop = (ev) => {
+  ev.preventDefault();
+  let elUnder = ev.target;
+  const classArray = elUnder.classList.value.split(' ');
+
+  if (
+    classArray.includes('fa-check')
+    || classArray.includes('checkbox')
+    || classArray.includes('to-do')
+  ) {
+    elUnder = elUnder.parentElement.parentElement;
+  } else if (
+    classArray.includes('fa-arrows-alt')
+    || classArray.includes('two')
+  ) {
+    elUnder = elUnder.parentElement;
+  }
+  prevNextIds.nextId = Number(elUnder.id);
+  prevNextIds.prevId = Number(dragStore.id);
+
+  if (prevNextIds.prevId < prevNextIds.nextId) {
+    elUnder.parentElement.insertBefore(dragStore, elUnder.nextElementSibling);
+  } else if (prevNextIds.prevId > prevNextIds.nextId) {
+    elUnder.parentElement.insertBefore(dragStore, elUnder);
+  }
+
+  const { id } = elUnder;
+
+  if (prevNextIds.prevId < prevNextIds.nextId) {
+    setTimeout(() => {
+      let elUnder = document.getElementById(id);
+      // document.getElementById(id).style.backgroundColor = "#4c3c3c";
+      const classArray = elUnder.classList.value.split(' ');
+      if (
+        classArray.includes('fa-check')
+        || classArray.includes('checkbox')
+        || classArray.includes('to-do')
+      ) {
+        elUnder = elUnder.parentElement.parentElement;
+      } else if (
+        classArray.includes('fa-arrows-alt')
+        || classArray.includes('two')
+      ) {
+        elUnder = elUnder.parentElement;
+      }
+
+      elUnder.style.backgroundColor = '#4c3c3c';
+      elUnder.style.color = '#fff';
+      Array.from(elUnder.childNodes).forEach((node) => {
+        if (node.nodeName === 'DIV') {
+          const nodes = node.childNodes;
+          Array.from(nodes).forEach((one) => {
+            one.style.backgroundColor = '#4c3c3c';
+            one.style.color = '#fff';
+          });
+        }
+        node.style.backgroundColor = '#4c3c3c';
+        node.style.color = '#fff';
+      });
+      setTimeout(() => {
+        // document.getElementById(id).style.backgroundColor = "#fff";
+        let elUnder = document.getElementById(id);
+        // document.getElementById(id).style.backgroundColor = "#4c3c3c";
+        const classArray = elUnder.classList.value.split(' ');
```

Comment:
You have some comments in the code. It will be more nice and more professional to remove them.

Reply:
Oh! my bad. I will work on all these, my friend. Thanks